openupgrade_scripts/scripts/sale/15.0.1.2/end-migration.py

```diff
@@ -0,0 +1,11 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    all_sale_order_lines = env["sale.order.line"].search(
+        [("product_packaging_id", "!=", False)]
```

Comment:
This should be restricted to quotations, as we don't want past orders to be modified on the migration. It's legit to do it for keeping a consistent state on pending ones:

```suggestion
        [("product_packaging_id", "!=", False), ('state', 'in', ['draft', 'sent'])]
```